services/autoscaling/tests/unit/test_modules_auto_scaling_core.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+import logging
+
+import pytest
+from fastapi import FastAPI
+from pytest_simcore.helpers.monkeypatch_envs import EnvVarsDict, setenvs_from_dict
+from simcore_service_autoscaling.modules.auto_scaling_core import (
+    _sorted_allowed_instance_types,
+)
+
+
+@pytest.fixture
+def with_empty_ec2_intances_allowed_types(
```

Comment:
normally this should allow for NO types at all.

Reply:
same as above :-) let's discuss offline